polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -87,7 +111,8 @@ final case class PolynoteConfig(
   spark: Map[String, String] = Map.empty,
   behavior: Behavior = Behavior(),
   security: Security = Security(),
-  ui: UI = UI()
+  ui: UI = UI(),
+  shared: Shared = Shared()
```

Comment:
Can we move this inside of `behavior` and call it `sharedPackages` (it would then be `shared_packages` in YAML)